recipe/meta.yaml

```diff
@@ -21,20 +21,17 @@ source:
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
-  skip: True  # [py<37]
+  skip: True  # [py<38]
   skip: True  # [linux and (s390x or ppc64le)]
 
 requirements:
   host:
-    - python
+    - python >=3.6
     - pip
-    - setuptools
-    - wheel
   run:
-    - python
+    - python >=3.6
     - pip    >=18.0
```

Comment:
Please remove whitespaces in `pip    >=18.0` it requires only one space

Reply:
will do

Reply:
`pip >=18.0` is still required for runtime because one of the addons uses it https://github.com/biolab/orange-canvas-core/blob/61fe8c11b92bbfd78d49757601c4796de1a1167c/orangecanvas/application/utils/addons.py#L554-L571

Reply:
wow, that's an excellent catch!